app/app/urls.py

```diff
@@ -28,7 +28,8 @@
 import retail.views
 import tdi.views
 from dashboard.router import router
-from sitemap import sitemaps
+from django.contrib.sitemaps.views import sitemap
+from . import sitemaps
```

Comment:
This breaks the sitemaps.  This should probably be `from .sitemaps import sitemaps`

Reply:
👍